### PR TITLE
New entry for PHP array_count_values function

### PR DESCRIPTION
**Pull request by Judis07**
### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/arrays/terms/array-count-values/array-count-values.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`array_count_values()`** is a function that counts the occurrences of values in an array. It accepts an input array and returns an associative array, where the keys are the unique values found in the array, and the corresponding values are the number of occurrences of each unique value.
+
+## Syntax
+
+```pseudo
+array_count_values($array)
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-count-values/array-count-values.md:20]:
```suggestion
$result = array_count_values($array)
```

File: content/php/concepts/arrays/terms/array-count-values/array-count-values.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+```
+
```
Comment (SSwiniarski) [inline content/php/concepts/arrays/terms/array-count-values/array-count-values.md:22]:
```suggestion

- `$array` is the array whose values are being counted.
- `$result` is the resultant array with counts.

```

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@Judis07 I did a first review and left a couple of comments, basically about building out the syntax section.

**Comment by Judis07** [comment]
@SSwiniarski suggested changes are now done. Please review.

**Review by caupolicandiaz — APPROVED** [review]
@Judis07 @SSwiniarski Just made some minor edits to the example intros. Looks good to merge

**Comment by SSwiniarski** [comment]
@Judis07 Congrats on the new entry. Live link here: https://www.codecademy.com/resources/docs/php/arrays/array-count-values

image

**Pull request merged**